server/tests/test_data/golden_menu_text.json

```diff
@@ -0,0 +1,122 @@
+    "Gross domestic product per capita",
+    "Gross domestic product growth rate",
+    "Gross national income per capita (purchasing power parity)",
+    "Gross national income (purchasing power parity)",
+    "Government debt",
+    "Government expenditures on education (% of government expenditure)",
+    "Government expenditures on education (% of GDP)",
+    "Government expenditures on military",
+    "Government expenditures on military (% of GDP)",
+    "Market capitalization of domestic companies",
+    "Market capitalization of domestic companies (% of GDP)",
+    "Inward remittance",
+    "Inward remittance (% of GDP)",
+    "Outward remittance",
+    "Median individual income",
+    "Individual income",
+    "Median household income",
+    "Household income",
+    "Worker wages (daily)",
+    "Worker wages (monthly)",
+    "Worker wages (monthly): urban vs rural",
+    "Worker wages (monthly): female vs male"
+  ],
+  [
+    "Life expectancy (years)",
+    "COVID-19 cumulative cases",
+    "COVID-19 cumulative deaths",
+    "Mortality events",
+    "Infant mortality rate",
+    "Infant mortality rate by gender",
+    "Wasting among children under 5",
+    "Wasting among children under 5 by gender",
+    "Severe wasting among children under 5",
+    "Severe wasting among children under 5 by gender",
+    "Health outcomes",
+    "Health behaviors",
+    "Prescribed drugs (quarterly)",
+    "Alcohol consumption per capita (annual)"
+  ],
+  [
+    "[Education] Rate of associate degree attainment by gender",
+    "[Education] Rate of bachelor's degree attainment by gender",
+    "[Education] Rate of graduate or professional degree attainment by gender",
+    "[Income] Gini index",
+    "[Income] Median household income by race",
+    "[Income] Median income by gender",
+    "[Poverty] Poverty rate by race",
+    "[Poverty] Poverty rate by gender",
+    "[Poverty] Urban vs rural poverty population",
+    "[Poverty] Urban vs rural homeless population",
+    "[Poverty] Unemployment rate by gender",
+    "[Poverty] Percentage of people without health insurance by race",
+    "[Poverty] Percentage of people without health insurance by gender"
+  ],
+  ["Total crime", "Violent crime", "Property crime", "Arson"],
+  ["Education attainment", "School enrollment"],
+  [
+    "Population",
+    "Population growth rate",
+    "Population density (people per km²)",
+    "Urban and rural population",
+    "Urban and rural households",
+    "Population by age",
+    "Population by gender",
+    "Population by race",
+    "Median Age",
+    "Median age by gender",
+    "Median age by race",
+    "Citizenship status",
+    "Marital status",
+    "Fertility rate",
+    "Literacy",
+    "Social Category",
+    "Religion"
+  ],
+  [
+    "Housing units",
+    "Housing units by householder race",
+    "Housing units by price",
+    "Housing units by date built",
+    "Housing units by rental status"
+  ],
+  [
+    "[Air] Air quality index",
```

Comment:
These shouldn't have the group in the titles after the string change?

Reply:
These are the actual title displayed. It's still nice to show them?

Reply:
I thought it'd be just a temporary thing to work around what we had. We can work on adding the group to the left-menu which might help.

I also missed that it's now being added in code below. I only noticed the change in the strings files.

Reply:
I reverted the change to make "topic" only an internal representation but don't show it in the UI. Later on we can add another level of menu from it if needed.